src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -62,9 +60,19 @@ function PolicyDistanceRatesPage({policy, route}: PolicyDistanceRatesPageProps)
     const customUnitRates: Record<string, Rate> = useMemo(() => customUnit?.rates ?? {}, [customUnit]);
 
     function fetchDistanceRates() {
-        Policy.openPolicyDistanceRatesPage(route.params.policyID);
+        Policy.openPolicyDistanceRatesPage(policyID);
     }
 
+    const dismissError = useCallback(
+        (item: RateForList) => {
+            if (item.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD) {
+                return;
+            }
```

Comment:
Why do we need this condition?

Reply:
It was to distinguish action that we should call. This is called when user presses on this X button to close the error

![image](https://github.com/Expensify/App/assets/23487612/bd6f6756-cab2-4f41-83a0-7a22f00552c7)

Reply:
Similar to the conversation below - Im not sure anymore how this part of error handling should work.

Reply:
<img width="869" alt="image" src="https://github.com/Expensify/App/assets/23487612/ecba83a7-2559-432a-bb38-25bdf804c169">
If whole thing should work as described then we have to keep this condition. We have to react in a different way depending on the scenario we're in

Reply:
However right know it has no purpose as delete is not implemented yet

Reply:
Yea, I commented above. We can have one method that clears the error and returns early if the pendinAction is update/delete. If the pendingAction is add, it also removes the item